DigitalLearningSolutions.Web/ViewModels/Support/SupportTicketsViewModel.cs

```diff
@@ -2,14 +2,26 @@
 {
     using DigitalLearningSolutions.Web.Models.Enums;
 
-    public class SupportTicketsViewModel : SupportViewModel
+    public class SupportTicketsViewModel : ISupportViewModel
     {
         public SupportTicketsViewModel(
             DlsSubApplication dlsSubApplication,
             SupportPage currentPage,
             string currentSystemBaseUrl
-        ) : base(dlsSubApplication, currentPage, currentSystemBaseUrl) { }
+        )
+        {
+            CurrentPage = currentPage;
+            DlsSubApplication = dlsSubApplication;
+            SupportSideNavViewModel = new SupportSideNavViewModel(currentSystemBaseUrl);
+            SupportTicketsIframeUrl = $"{currentSystemBaseUrl}/tracking/tickets?nonav=true";

```

Comment:
Done.

Reply:
Looks like it's still here 😛

Reply:
Sorry, I've assumed this was related to the other comment. I know what you mean now.
I'm not sure if there is a use for this value anywhere else but given we now have a helper where this could fit I will drop it in.

Reply:
And now it's gone.